.github/workflows/cover.yaml

```diff
@@ -13,12 +13,31 @@
 # limitations under the License.
 
 name: code coverage
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
 
 jobs:
   build:
+    if: "${{ github.event.action != 'labeled' || github.event.label.name == 'tests: run' }}"
```

Comment:
@kurtisvg one side effect of this logic currently is that if a label other than `tests: run` is added to the PR after successful tests, the new `labeled` event trigger will cause the already successful github actions for tests to move to the `skipped` state and require the `tests:run` label to be added in order to re-run the required tests. I'm not sure how big of a deal this is for us as we don't usually label PRs after creation?